.github/workflows/build_pkg.yml

```
@@ -1,18 +1,24 @@
-name: "Build d-SEAMS"
+name: "Build d-SEAMS and test"
 on: [push, pull_request]
 jobs:
   build_pkg:
     runs-on: ubuntu-latest
     steps:
-    - uses: actions/checkout@v2.3.4
-    - uses: cachix/install-nix-action@v16
-      with:
-        nix_path: nixpkgs=channel:nixos-unstable
-    - uses: cachix/cachix-action@v10
-      with:
-        name: dseams
-        signingKey: '${{ secrets.CACHIX_SIGNING_KEY }}'
-        authToken: '${{ secrets.CACHIX_AUTH_TOKEN }}'
-    - run: nix-env -if .
-    - run: yodaStruct -c lua_inputs/config.yml
-    - run: echo OK
+      - uses: actions/checkout@v3
+
+      - name: Install Conda environment from environment.yml
+        uses: mamba-org/provision-with-micromamba@main
+
+      - name: Build application with cmake
+        shell: bash -l {0}
+        run: |
+          mkdir bbdir
```

```suggestion
        run: |
          luarocks install luafilesystem
          mkdir bbdir
```

As documented in the `readme`.